lib/game.rb

```diff
@@ -0,0 +1,131 @@
+            card33 = Card.new(:heart, '10', 10),
+            card34 = Card.new(:club, '10', 10),
+            card35 = Card.new(:diamond, '10', 10),
+            card36 = Card.new(:spade, '10', 10),
+            card37 = Card.new(:heart, 'Jack', 11),
+            card38 = Card.new(:club, 'Jack', 11),
+            card39 = Card.new(:diamond, 'Jack', 11),
+            card40 = Card.new(:spade, 'Jack', 11),
+            card41 = Card.new(:heart, 'Queen', 12),
+            card42 = Card.new(:club, 'Queen', 12),
+            card43 = Card.new(:diamond, 'Queen', 12),
+            card44 = Card.new(:spade, 'Queen', 12),
+            card45 = Card.new(:heart, 'King', 13),
+            card46 = Card.new(:club, 'King', 13),
+            card47 = Card.new(:diamond, 'King', 13),
+            card48 = Card.new(:spade, 'King', 13),
+            card49 = Card.new(:heart, 'Ace', 14),
+            card50 = Card.new(:club, 'Ace', 14),
+            card51 = Card.new(:diamond, 'Ace', 14),
+            card52 = Card.new(:spade, 'Ace', 14)]
+    
+        full_deck.shuffle!
+        deck1 = Deck.new([])
+        deck2 = Deck.new([])
+    
+        player1 = Player.new("Megan", deck1)
+        player2 = Player.new("Aurora", deck2)
+    
+        26.times do
+            player1.deck.cards << full_deck.shift
+        end
+    
+        26.times do
+            player2.deck.cards << full_deck.shift
+        end
+
+         turn_number = 0
+            until turn_number == 10000
+                turn_number += 1
+                turn = Turn.new(player1, player2)
+                if player1.deck.cards.length == 0 || player2.deck.cards.length == 0
+                    if player1.deck.cards.length == 0
+                        winner = player1
+                    else 
+                        winner = player2
+                    end
+                    p "*~*~*~* #{winner.name} has won the game outright! *~*~*~*"
+                    exit
+                end
+                current_turn = turn.type
+                winner = turn.winner
+                    if current_turn == :loss
+                        p "*~*~*~* #{winner.name} has won the game by default! *~*~*~*"
+                        exit
+                    elsif current_turn == :mutually_assured_destruction
+                        turn.pile_cards
+                        p "Turn #{turn_number.to_s}: *mutually assured destruction* 6 cards removed from play"
+                    elsif current_turn == :war                    
+                        turn.pile_cards
+                        p "Turn #{turn_number.to_s}: WAR - #{winner.name} won #{turn.spoils_of_war.length} cards"
+                    elsif current_turn == :basic
+                        turn.pile_cards
+                        p "Turn #{turn_number.to_s}: #{winner.name} won #{turn.spoils_of_war.length} cards"
+                    end          
+                turn.award_spoils_of_war(winner)
+            end                   
+    end
```

Comment:
I don't see anything amiss with this `start` method. I will mention a few things though:
* This method is super long and should be broken down into smaller helper methods
* The indentation in this method is a little off